src/input/mod.rs

```diff
@@ -152,22 +150,30 @@ impl<'a> InputFile<'a> {
 
 
     /// Reads image data into the given FrameBufferMut.
+    ///
+    /// The passed FrameBufferMut must match the image's resolution exactly, and
```

Comment:
Yeah, I did a test, and the C++ API does expect them to match (throws an exception if they don't).

I'll add that. :-)